paddle/fluid/framework/ir/mkldnn/cpu_quantize_pass.cc

```diff
@@ -202,23 +207,45 @@ void CPUQuantizePass::DequantizeOutput(Graph* g, Node* op, Node* output,
   if (!scale_attr_name.empty()) op->Op()->SetAttr(scale_attr_name, scale);
 }
 
+bool CPUQuantizePass::AreScalesPresentForVarNames(
+    std::vector<std::string> names) const {
+  auto& scales = Get<VarQuantScale>("quant_var_scales");
+  bool present = true;
+  for (auto name : names) {
+    if (scales.find(name) == scales.end()) {
+      present = false;
+      LogScaleIsMissingForVarName(name);
+    }
+  }
+  return present;
+}
+
 bool CPUQuantizePass::AreScalesPresentForNodes(
-    const Node* op_node, std::initializer_list<Node*> nodes) const {
+    std::initializer_list<Node*> nodes) const {
   auto& scales = Get<VarQuantScale>("quant_var_scales");
   bool present = true;
   for (auto node : nodes) {
     if (scales.count(node->Name()) == 0) {
       present = false;
-      LogScaleIsMissingForVar(node);
+      LogScaleIsMissingForVarNode(node);
     }
   }
   return present;
 }
 
+std::pair<bool, LoDTensor> CPUQuantizePass::GetScaleDataByName(
+    const std::string& name) const {
+  auto& scales = Get<VarQuantScale>("quant_var_scales");
+  return scales[name];
```

Comment:
Correct if I'm wrong please, function name suggests getter so maybe **at** function would be better in that case as [] operator has some side effects ?

Reply:
OK, done.

Reply:
OK